packages/svelte2tsx/src/svelte2tsx.ts

```diff
@@ -452,15 +452,18 @@ function processInstanceScriptContent(str: MagicString, script: Node): InstanceS
         }
 
         if (ts.isExportDeclaration(node)) {
-            for (const ne of node.exportClause.elements) {
-                if (ne.propertyName) {
-                    addExport(ne.propertyName, ne.name);
-                } else {
-                    addExport(ne.name);
+            const { exportClause } = node;
```

Comment:
Just curious: For what is this update to the code needed?

Reply:
Ah, nevermind, it's for getting rid of the type errors.

Reply:
This just adds another check if the `exportClause` is a NamedExports so that typescript won't complain that `elements` doesn't exist on type `NamespaceExport`.